src/commands/support/hud.ts

```diff
@@ -0,0 +1,14 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+export const hud: CommandDefinition = {
+    name: ['hud'],
```

Comment:
```suggestion
    name: 'hud',
```
no need to have an array for a single alias